libsemigroups_cppyy/froidure_pin.py

```diff
@@ -22,4 +20,25 @@ def FroidurePin(*args):
     froidure_pin_type = cppyy.gbl.libsemigroups.FroidurePin(types.pop())
     detail.unwrap(froidure_pin_type, froidure_pin_type.factorisation, lambda x:
             list(x))
+    detail.unwrap_int(froidure_pin_type, froidure_pin_type.left)
+    detail.unwrap_int(froidure_pin_type, froidure_pin_type.right)
+    detail.unwrap_int(froidure_pin_type, froidure_pin_type.nr_generators)
     return froidure_pin_type(gens)
+
+def right_cayley(S):
+    edges = []
+    for i in range(S.size()):
+        for j in range(S.nr_generators()):
+            edges.append((str(S.factorisation(S[i])),str(S.factorisation(S.right(i,j)))))
+    G=networkx.DiGraph()
+    G.add_edges_from(edges)
+    return(networkx.draw(G, with_labels = True))
```

Comment:
I think this should just return G, not the picture of G. Also might it not be better to create G at the start of the function, and then add edges to it in the nested loops, rather than creating the list edges, which is almost immediately thrown away? I'm not sure about the capabilities of networkx, perhaps this isn't possible.